.github/workflows/test.yml

```
@@ -93,6 +93,19 @@ jobs:
       - name: Checkout code
         uses: actions/checkout@v4
 
+      # This is a temporary solution.
+      # Once ubuntu-latest includes the latest libxml2 version, this step can be removed.
```

Please can you expand "latest" into a version number that we can check in the future. It seems that there is a minimum version where this works from. When `ubuntu-latest` has this version, we may want to use the version before that so that we continue testing both sides of this change.
```suggestion
      # Once ubuntu-latest includes libxml2 version X.Y.Z or newer, this step can be removed.
```